### PR TITLE
feat(charts): bridge-withdrawer, smoke test, various chart improvements

### PR DESCRIPTION
**Pull request by joroshiba**
## Summary
Adds the bridge withdrawer chart, updates smoke test to use it, improves the release process of charts, and has a slew of other chart upgrades to bring most of charts closer to same standard.

## Background
We want to make sure the bridge withdrawal path works across changes, need chart for running in production.

## Changes
- fix sequencer query which was not updated with change to asset from asset id
- Add an `evm-bridge-withdrawer` helm chart
- Update smoke test to utilize bridge withdrawer path
- Update `deploy.just` to default to using the deployed bridge and have init commands.
- Improve the release process of charts to [astriaorg/charts](https://github.com/astriaorg/charts). Now includes the README from `/charts` and the `/dev` directory so you can utilize just the charts repo for lighter weight deployment.
- Update charts to have liveness, startup, and readyness probes where appropriate
- Remove some unused values files, service fields
- Remove settings which should usually be changed from evm-rollup and sequencer `values.yaml` in favor of specifying in the dev override files.
- Fix the CREATE2 script in alloc of evm rollup
- Updated the default kind image, and cilium. Disable hubble by default
- update local markdown lint to not hit any bridge contract sub module code
- Added just convenience just commands for curling eth json rpc

## Testing
Various local testing plus the CI/CD updated smoketest

## Breaking Changelist
- New releases of many charts due to removal of default values.

### CODE REVIEW

**Review by aajimal — DISMISSED** [review]

File: crates/astria-bridge-withdrawer/ethereum/README.md [hunk 1]
```diff
@@ -36,3 +36,68 @@ Call `withdrawToOriginChain` in `AstriaWithdrawer.sol`:
...
+
+1. Note the withdraw TX hash and get the raw tx code for it:
+
+    ```sh
+    > just evm-get-raw-transaction <withdraw-tx-hash>
+    <new-bridge-tx>
+    ```
+
+1. Update the `bridge_tx_bytes` and `bridge_tx_hash` fields in
+`[repo-root]/charts/deploy.just` with the new raw tx bytes and hash repectfully.
```
Comment (noot) [inline crates/astria-bridge-withdrawer/ethereum/README.md:103]:
```suggestion
`[repo-root]/charts/deploy.just` with the new raw tx bytes and hash respectfully.
```

File: crates/astria-bridge-withdrawer/ethereum/README.md [hunk 2]
```diff
@@ -36,3 +36,68 @@ Call `withdrawToOriginChain` in `AstriaWithdrawer.sol`:
...
+1. Get the contract address deployed:
+
+    ```sh
+    > just evm-get-deployed-contract-code <deployed-contract-address>
+    <new-contract-code>
+    ```
+
+1. Update the `genesisAlloc` section in `[repo-root]/dev/values/rollup/dev.yml`
+with the new contract code.
+1. Submit a withdraw TX to the new contract:
```
Comment (noot) [inline crates/astria-bridge-withdrawer/ethereum/README.md:85]: this step would only be needed if the `withdrawToSequencer` function sig in the contract changes

**Review by noot — DISMISSED** [review]

**Review by steezeburger — DISMISSED** [review]

File: crates/astria-bridge-withdrawer/ethereum/out/AstriaWithdrawer.sol/AstriaWithdrawer.json
Comment (SuperFluffy) [inline crates/astria-bridge-withdrawer/ethereum/out/AstriaWithdrawer.sol/AstriaWithdrawer.json:1]: empty file?

File: crates/astria-cli/src/commands/sequencer.rs [hunk 3]
```diff
@@ -82,7 +89,8 @@ pub(crate) fn create_account() {
     // https://github.com/astriaorg/astria/issues/594
     println!("Private Key: {private_key_pretty:?}");
     println!("Public Key:  {public_key_pretty:?}");
-    println!("Address:     {address_pretty:?}");
+    println!("Address:     \"{astria_address}\"");
+    println!("Address ugly: {address_pretty:?}");
```
Comment (SuperFluffy) [inline crates/astria-cli/src/commands/sequencer.rs:93]: hahaha

**Review by SuperFluffy — APPROVED** [review]

File: dev/values/validators/all.yml
Comment (quasystaty1) [inline dev/values/validators/all.yml:1]: nit: newline

File: charts/evm-bridge-withdrawer/templates/configmaps.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: evm-bridge-withdrawer-env
+  namespace: {{ include "evm-bridge-withdrawer.namespace" . }}
+data:
+  ASTRIA_BRIDGE_WITHDRAWER_LOG: "astria_bridge_withdrawer=debug"
+  ASTRIA_BRIDGE_WITHDRAWER_API_ADDR: "0.0.0.0:{{ .Values.ports.healthAPI }}"
+  ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_COMETBFT_ENDPOINT: "{{ .Values.config.sequencerCometBftEndpoint }}"
```
Comment (quasystaty1) [inline charts/evm-bridge-withdrawer/templates/configmaps.yaml:9]: nit: maybe change `sequencerCometBftEndpoint` to `sequencerCometbftEndpoint` to keep consistency with sequencer chart

**Review by quasystaty1 — APPROVED** [review]

File: crates/astria-sequencer/src/service/info/mod.rs [hunk 5]
```diff
@@ -60,8 +60,8 @@ impl Info {
             .context("invalid path: `asset/denom/:id`")?;
         query_router
             .insert(
-                "asset/allowed_fee_asset_ids",
-                crate::asset::query::allowed_fee_asset_ids_request,
+                "asset/allowed_fee_assets",
```
Comment (SuperFluffy) [inline crates/astria-sequencer/src/service/info/mod.rs:63]: To confirm our offline discussion: this change is correct, just overlooked this in the big refactor to remove IDs

**Review by SuperFluffy — APPROVED** [review]
Approving the query path renaming. That was my oversight.

Sidenote: that abci query will likely return ibc-prefixed denoms right now. That's probably somewhat questionable for the users and might be changed to return the full trace-prefixed denoms in the future (we make no guarantees for this and changing it will not be a breaking).

**Pull request merged**